.bazelrc

```diff
@@ -1,11 +1,6 @@
-##################
-# Startup config #
-##################
-
-# The memory usage of the java process are skyrocketing with journey-maps build (bazel bug?).
-# Limit it to 2g make the build very fast and without error.
-# It's related with https://github.com/angular/angular/issues/46965
-startup --host_jvm_args=-Xmx2g
+#########
+# Tests #
```

Comment:
# RAM Issues #
...

# To prevent 'Worker process did not return a WorkResponse' maybe caused by a high memory consumption the recommandation on https://bazel.build/docs/memory-saving-mode is tested
# It's related with https://github.com/angular/angular/issues/46965